### ISSUE TITLE
Total social integration of the mappers

### ISSUE THREAD

**Issue by yopaseopor** [mapillary/mapillary_issues#98]
What about integration with Twitter / Facebook / Google + / Pinterest / Instagram / LinkedIn / Flickr ...

**Comment by gyllen** [comment]
Sharing is added to images

**Comment by yopaseopor** [comment]
Yes but also for Mapillarers like the activity tab data, is there any RSS
or something like that to attach taht data automatically to Twitter,
Facebook, etc. ?
